src/k8-client/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2021"
 name = "k8-client"
-version = "12.0.1"
```

Comment:
since pub API are changed, this should be bump to `13.0.0`

Reply:
bumped